src/routers/user.ts

```diff
@@ -152,6 +153,12 @@ export const userRouter = () => {
     checkUserSignInRequirementsMiddleware,
     issueSessionOrRedirectController
   );
+  userRouter.get(
+    '/send-magic-link-again',
+    csrfProtectionMiddleware,
+    checkEmailInSessionMiddleware,
+    getSendMagicLinkAgainController
```

Comment:
J'ai mis les mêmes protections que sur la page sign-in normale, puisque c'est sur celle-là qu'à l'origine on a le bouton "envoyer le lien".